messagingpush/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,15 @@
                 <action android:name="io.customer.messagingpush.PUSH_ACTION" />
             </intent-filter>
         </receiver>
+        <!-- Broadcast receiver for listening to push events from GCM -->
```

Comment:
I think we should expand these comments to explain why we decided to add this broadcast receiver to the code base. For future readers of the code wondering why there seems to be 2 implementations of push delivered metrics in the SDK.

Reply:
Added more comments to it